js/main.js

```diff
@@ -58,6 +59,24 @@ $searchBar.addEventListener('submit', function (event) {
   responseGET(getRecipeIDs, 'filter.php?c=' + searched); // category
 });
 
+gallery.addEventListener('click', function (event) {
+  if (event.target.matches('.bookmark')) {
```

Comment:
I would just move this conditional to the `handleBookmark` function as a guard so you can remove the need for an anonymous function here.

Reply:
Discussed with Cody to allow this formatting for future use of other events.